src/coreclr/gc/gc.cpp

```diff
@@ -23360,6 +23404,21 @@ size_t gc_heap::get_total_gen_estimated_reclaim (int gen_number)
     return total_estimated_reclaim;
 }
 
+size_t gc_heap::get_total_gen_size (int gen_number)
+{
+#ifdef MULTIPLE_HEAPS
+    size_t size = 0;
+    for (int hn = 0; hn < gc_heap::n_heaps; hn++)
+    {
+        gc_heap* hp = gc_heap::g_heaps[hn];
+        size += hp->generation_sizes (hp->generation_of (gen_number));
```

Comment:
this should use `generation_size`, not `generation_sizes`

Reply:
Yup, thanks for catching!